articles/aks/keda-deploy-add-on-cli.md

```diff
@@ -0,0 +1,537 @@
+                      type: integer
+                    status:
+                      description: HealthStatusType is an indication of whether the
+                        health status is happy or failing
+                      type: string
+                  type: object
+                type: object
+              lastActiveTime:
+                format: date-time
+                type: string
+              originalReplicaCount:
+                format: int32
+                type: integer
+              pausedReplicaCount:
+                format: int32
+                type: integer
+              resourceMetricNames:
+                items:
+                  type: string
+                type: array
+              scaleTargetGVKR:
+                description: GroupVersionKindResource provides unified structure for
+                  schema.GroupVersionKind and Resource
+                properties:
+                  group:
+                    type: string
+                  kind:
+                    type: string
+                  resource:
+                    type: string
+                  version:
+                    type: string
+                required:
+                - group
+                - kind
+                - resource
+                - version
+                type: object
+              scaleTargetKind:
+                type: string
+            type: object
+        required:
+        - spec
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    status: "True"
+    type: NamesAccepted
+  - lastTransitionTime: "2022-06-08T10:31:07Z"
+    message: the initial names have been accepted
+    reason: InitialNamesAccepted
+    status: "True"
+    type: Established
+  storedVersions:
+  - v1alpha1
+  ```
+
+KEDA provides multiple customization options. The KEDA add-on enables the basic common configuration. If you have requirement to run with some  custom configurations for example:  number of KEDA pods running, namespaces that should be watched, logging level, etc. then you may edit the KEDA yaml manually and deploy it. Remember the customizations would be managed by you. 
+
+## Disable KEDA add-on from your AKS cluster
+
+When you no longer need KEDA add-on in the cluster, use the `az aks update` command with--disable-keda option. This will disable KEDA workload auto-scaler
+
+```azurecli-interactive
+az aks update \
+  --resource-group myResourceGroup \
+  --name myAKSCluster \
+  --disable-keda 
+```
+
+## Troubleshooting guide
+
+If managed KEDA is installed in an AKS cluster with user installed KEDA, then Managed KEDA overrides user's KEDA. If user's KEDA has some customizations added, they'll get lost once managed KEDA replaces it. The installation will complete without error. User's auto-scaling should most probably work fine as before. There will be two copies of KEDA resources present in the AKS cluster and only one of them will be doing the job of scaling.
```

Comment:
What about this?
```suggestion
While Kubernetes only allows one metric server to be installed, you can in theory install KEDA multiple times. However, this is not recommended given only one installation will work.

When the KEDA add-on is installed in an AKS cluster, the previous installation of open-source KEDA will be overridden and the add-on will take over.

This means that the customization and configuration of the self-installed KEDA deployment will get lost and no longer be applied.

While there is a possibility that the existing autoscaling will keep on working, there is a risk given it will be configured differently and will not support features such as managed identity.

It is recommended to uninstall existing KEDA installations before enabling the KEDA add-on given the installation will succeed without any error.
```